addOns/network/src/main/java/org/zaproxy/addon/network/GlobalExclusionsOptions.java

```diff
@@ -186,8 +186,33 @@ protected void updateConfigsImpl(int fileVersion) {
                         ((HierarchicalConfiguration) getConfig()).configurationsAt(EXCLUSION_KEY);
                 if (fields.isEmpty()) {
                     setGlobalExclusions(DEFAULT_GLOBAL_EXCLUSIONS);
-                    break;
                 }
+                break;
+            case 1:
+                List<HierarchicalConfiguration> fields1 =
+                        ((HierarchicalConfiguration) getConfig()).configurationsAt(EXCLUSION_KEY);
+                globalExclusions = new ArrayList<>(fields1.size());
+                List<String> defaultNames = new ArrayList<>(DEFAULT_GLOBAL_EXCLUSIONS.size());
+                DEFAULT_GLOBAL_EXCLUSIONS.forEach(e -> defaultNames.add(e.getName()));
+                for (HierarchicalConfiguration sub : fields1) {
+                    try {
+                        String value = sub.getString(GLOBAL_EXCLUSION_VALUE_KEY, "");
+                        String name = sub.getString(GLOBAL_EXCLUSION_NAME_KEY, "");
+                        if (validateGlobalExclusionPattern(value)) {
+                            if (defaultNames.contains(name)) {
```

Comment:
Would be better to check the values rather than the names, if the user changed the value we should not be tampering with it. This will also miss the name that was changed in this version.